modules.d/62bluetooth/module-setup.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/sh
+# This file is part of dracut.
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+# Prerequisite check(s) for module.
+check() {
+  # If the binary(s) requirements are not fulfilled the module can't be installed
+  require_binaries /usr/libexec/bluetooth/bluetoothd || return 1
```

Comment:
Can also be /usr/lib/bluetooth/bluetoothd, depending on the distribution.

Reply:
Should work in the latest revision of this pull request.